src/Installers/Windows/WindowsHostingBundle/thm.wxl

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
-  <String Id="Caption">[WixBundleName] Setup</String>
-  <String Id="Title">[BundleNameShort]</String>
-  <String Id="SubTitle">[BundleNameSub]</String>
+  <String Id="Caption"><!--_locComment_text="{Locked='[WixBundleName]'"-->[WixBundleName] Setup</String>
```

Comment:
missing the closing }

Reply:
Yeah, now I see that I did that a lot. Will fix **Real Soon Now:tm:**